flexer/cli.py

```diff
@@ -72,6 +73,38 @@ def list(ctx):
         )
 
 
+@cli.command(name='new')
+@click.option('--name',
+              required=True,
+              help="A name for the new module")
+@click.option('--event-source',
+              required=True,
+              type=click.Choice(EVENT_SOURCES),
+              help="The event source for the module")
+@pass_context
+def new_module(ctx, name, event_source):
+    """
+    Create a new nFlex module.
+    """
+
+    module_type = ModuleTemplate.get_module_type(event_source)
+    click.echo(
+        'Creating a new {} module...'.format(module_type)
+    )
+
+    templates_dir = os.path.join(
+        os.path.dirname(__file__),
+        "templates",
+        module_type
+    )
```

Comment:
BTW, I use PyCharm - it's always linting (PEP8), but I don't normally follow the maximum 80 chars limit. But I'm happy to do so here and will sort that out now :)